src/Snakemake/rules/QC/picard.smk

```diff
@@ -0,0 +1,60 @@
+localrules:
+    touchBatch,
+
+
+rule picardHsMetrics:
+    input:
+        bam="DNA_bam/{sample}-ready.bam",
+        intervals=config["bed"]["intervals"],
+    output:
+        "qc/{sample}/{sample}.HsMetrics.txt",
+    log:
+        "logs/qc/picard/HsMetrics/{sample}.log",
+    singularity:
+        config["singularity"]["picard"]
+    shell:
+        "(java -Xmx4g -jar /opt/conda/share/picard-2.20.1-0/picard.jar CollectHsMetrics BAIT_INTERVALS={input.intervals} TARGET_INTERVALS={input.intervals} INPUT={input.bam} OUTPUT={output}) &> {log}"
+
+
+rule picardInsertSize:
+    input:
+        bam="DNA_bam/{sample}-ready.bam",
+    output:
+        txt="qc/{sample}/{sample}.insert_size_metrics.txt",
+        pdf="qc/{sample}/{sample}.insert_size_histogram.pdf",
+    log:
+        "logs/qc/picard/InsertSize/{sample}.log",
+    singularity:
+        config["singularity"]["picard"]
+    shell:
+        "(java -Xmx4g -jar /opt/conda/share/picard-2.20.1-0/picard.jar CollectInsertSizeMetrics INPUT={input.bam} O={output.txt} H={output.pdf}) &> {log}"
+
+
+rule PicardAlignmentSummaryMetrics:
+    input:
+        bam="DNA_bam/{sample}-ready.bam",
+        ref=config["reference"]["ref"],
+    output:
+        "qc/{sample}/{sample}.alignment_summary_metrics.txt",
+    log:
+        "logs/qc/picard/AlignmentSummaryMetrics/{sample}.log",
+    singularity:
+        config["singularity"]["picard"]
```

Comment:
maybe change to 
config["singularity"].get("picard", config["singularity"].get("default", "")) and add
```
- picard=2.25.0
```
 to  https://github.com/clinical-genomics-uppsala/Twist_DNA/blob/develop/dockerfiles/somatic/environment.yml 

and update https://github.com/clinical-genomics-uppsala/Twist_DNA/blob/develop/dockerfiles/somatic/VERSION with new version (0.0.15)

Reply:
done